src/js/services/optimizely.js

```diff
@@ -147,15 +148,15 @@ class OptimizelyClient {
 // trackExperimentViewed checks if we alredy have sent the Experiment Viewed
 // event to segment/amplitude by looking into the localstorage.
 // If not, it builds the properties needed and call trackAction with it
-const trackExperimentViewed = (
+export function trackExperimentViewed(
   orgId,
   experimentKey,
   experimentContainer,
   experimentId,
   variationName,
   variationId,
   userId,
-) => {
+) {
```

Comment:
could we also do a `export const trackExperimentViewed = (parameters.....) => {}` here?

Reply:
Same comment as above.